integreat_cms/cms/fixtures/test_data.json

```diff
@@ -1695,7 +1695,7 @@
       "title": "Alltag",
       "slug": "alltag",
       "status": "PUBLIC",
-      "content": "<a href=\"https://integreat-app.de/\">G&uuml;ltiger Link</a><br><a href=\"https://integreat.app\">Ungepr&uuml;fter Link</a><br><a href=\"https://invalid.integreat.app/\">Ung&uuml;ltiger Link</a><br><a href=\"https://invalid2.integreat.app/\">Ignorierter Link</a><br><br><a href=\"https://integreat.app/augsburg/de/willkommen/\">Interner Link</a>",
+      "content": "<a href=\"https://integreat-app.de/\">G&uuml;ltiger Link</a><br><a href=\"https://integreat.app\">Ungepr&uuml;fter Link</a><br><a href=\"https://invalid.integreat.app/\">Ung&uuml;ltiger Link</a><br><a href=\"https://invalid2.integreat.app/\">Ignorierter Link</a><br><a href=\"https://integreat.app/augsburg/de/willkommen/\">Interner Link</a><br><a href=\"https://integreat-app.de/\">Ein langer Link zum Testen des Verhaltens von Links im PDF Export. Text mit Leerzeichen sollte gebrochen werden, W&#246;rter mit mehr als 50 Zeichen sollten hingegen gek&#252;rzt werden. www.dashieristeinsehrlangerlinkalszusammenh&#228;ngendezeichenkette.de</a>",
```

Comment:
Was it intentional that the long text and `www.dashieristeinsehrlangerlinkalszusammenh&#228;ngendezeichenkette.de` are one and the same link? Or did you mean to create two different links?

Reply:
This was intentional. I did not see a reason to create a new link to test this behaviour as it is just about the text content and not about the actual link reference. However, I could of course also split it, if you think this is a better solution.

Reply:
Alright, no that's fine!